news/service/views.py

```diff
@@ -0,0 +1,37 @@
+from db import fetch_n_clusters_after_time, fetch_n_clusters_before_time
+from flask import current_app as app, request, jsonify
+from serializers import cluster_cursor_to_dict
+
+LOGGER = app.logger
+
+
+@app.route('/', methods=['GET'])
+def root():
+    return jsonify({'message': 'Graphelier News'}), 200
+
+
+@app.route('/news_clusters/<start_timestamp>', methods=['GET'])
+def news_clusters(start_timestamp):
+    direction = int(request.args.get('direction', '1'))
+    quantity = int(request.args.get('quantity', '10'))
+    try:
+        timestamp = int(start_timestamp)
+    except ValueError as e:
+        LOGGER.error(e)
+        return jsonify({'message': 'Given timestamp is badly formed: {}'.format(start_timestamp)}), 400
+
+    query_result = fetch_n_clusters_after_time(timestamp, quantity+1) if direction == 1 \
```

Comment:
why +1 on the quantity?

Reply:
I want to know the timestamp of the (last element + 1) of the query such that the user can know which timestamp to request next in terms of paging (`next_timestamp`).

Basically the user holds and updates the upper `next_timestamp` and `lower next_timestamp` for paging on client side

Reply:
but they would be able to do that with the first and last element they have on their end no?

Reply:
then they'll receive duplicated values for those elements when requesting with their respective timestamps

Reply:
oh i'm doing a `lt` / `gt`, shouldn't happen right ?

Reply:
that shouldn't be the case because the queries are lt/gt and not lte/gte